PIL/Image.py

```diff
@@ -1519,7 +1519,7 @@ def putpixel(self, xy, value):
             return self.pyaccess.putpixel(xy, value)
         return self.im.putpixel(xy, value)
 
-    def resize(self, size, resample=NEAREST):
+    def resize(self, size, resample=NEAREST, roi=None):
```

Comment:
Another possible name is `box=None` like in `Image.crop()`.

I would not refuse help with English docstring. My suggestion:

The region of interest for the source image. A (left, upper, right, lower)-tuple of floats. If not provided, the entire source image is used.